### DISCUSSION TITLE
`called Option::unwrap() on a None value` Error

### DISCUSSION THREAD

**Discussion by amirAlamian** [prisma/prisma#9971] [Q&A] [answered]
Hi Im Working on migration from MySQL  to MongoDB. Im using prisma as my ORM. When I want to Create some Mock data for my application, Prisma crashes and throws this error
```
thread 'tokio-runtime-worker' panicked at 'called `Option::unwrap()` on a `None` value', query-engine/connectors/mongodb-query-connector/src/root_queries/write.rs:301:74
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

```
I think its because of multiple connections to MongoDB when I send many queries. Im using a for loop to insert data into the one of my collections. 
Any thoughts?

**Comment by TasinIshmam** [comment, marked as answer]
Hey @amirAlamian :wave:  

Could you provide more information about the problem? The following would help

- The code/query causing the issue
- Your Prisma schema
- Prisma version (All information from `npx prisma --version`) 
- Your operating system
- Instructions on how to replicate the problem 
- Anything else that you might think might be relevant

**Reply by TasinIshmam** [reply]
Hey @amirAlamian  any updates on this? :eyes:

**Reply by TasinIshmam** [reply]
Marking as answered due to lack of updates (feel free to reopen if you have any updates @amirAlamian)

**Reply by janpio** [reply]
Generally this sounds more like a bug, so we would prefer a bug issue to be opened with all the information the bug template asks for.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
